alf/metrics/metrics.py

```diff
@@ -56,7 +56,7 @@ def append(self, value):
             Nothing
         """
         position = torch.remainder(self._head, self._max_len)
-        self._buffer[position] = value
+        self._buffer.scatter_(0, position, value)
```

Comment:
"self._buffer[position] = value" is more readable

Reply:
Done

Reply:
Done